Examples/php/pragmas/runme.php

```diff
@@ -2,4 +2,5 @@
 
 require "example.php";
 
+echo "Version - " . ((new ReflectionExtension('example'))->getVersion());
 ?>
```

Comment:
Thanks for sorting out the tests - aside from the "missing newline at end of file" nit, those look good.